Makefile

```diff
@@ -96,6 +107,16 @@ build-macos: $(DOTHERSIDE) $(STATUSGO) src/nim_status_client.nim | deps
 	echo -e $(BUILD_MSG) "$@" && \
 		$(ENV_SCRIPT) nim c -d:nimDebugDlOpen -L:$(STATUSGO) -d:ssl -L:-lm -L:"-framework Foundation -framework Security -framework IOKit -framework CoreServices" $(NIM_PARAMS) -L:$(DOTHERSIDE) --outdir:./bin src/nim_status_client.nim
 
+build-windows: $(DOTHERSIDE) $(STATUSGOWINDOWS) src/nim_status_client.nim | deps
+	echo -e $(BUILD_MSG) "$@" && \
+		$(ENV_SCRIPT) nim c -d:mingw --cpu:amd64 -d:nimDebugDlOpen \
```

Comment:
Why do you need these Nim defines and a CPU override?

Reply:
`--cpu:amd64 -d:nimDebugDlOpen`
Not needed. I had added `nimDebugDIOpen` before when I was having an issue with a the .AppImage not loading some libraries and forgot to remove it. And the cpu override was left from back when I was testing generating an output for i386